packages/keycloak-user-management/src/components/CreateEditUser/index.tsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { Col, notification, Row } from 'antd';
+import { RouteComponentProps } from 'react-router';
+import * as Yup from 'yup';
+import { Store } from 'redux';
+import { connect } from 'react-redux';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import { HeaderBreadCrumb } from '../HeaderBreadCrumb';
+import {
+  fetchKeycloakUsers,
+  KeycloakUser,
+  makeKeycloakUsersSelector,
+  reducer as keycloakUsersReducer,
+  reducerName as keycloakUsersReducerName,
+  getAccessToken,
+} from '@opensrp/store';
+import { KeycloakService } from '@opensrp/keycloak-service';
+import Ripple from '../Loading';
```

Comment:
I am completely against duplicating the Loading component withing this package!

Reply:
This could be addressed via https://github.com/OpenSRP/web/pull/58